src/components/Articles/ReadArticle.js

```diff
@@ -52,7 +53,10 @@ const ReadArticle = ({ article }) => (
           <div className="col-5">
             <div id="buttons" className={getAuthor(article.author.username)}>
               <input type="button" value="Edit" className="btn-edit" />
-              <input type="button" value="Delete" className="btn-delete" />
+              <input type="button" value="Delete" className="btn-delete" data-toggle="modal" data-target="#exampleModal" />
+            </div>
+            <div>
+              <DeleteModal id="exampleModal" slug={slug} />
```

Comment:
Awesome work @mendozabree. However, do we need an `id` here?

Reply:
Oh yes, thank you for noticing that. And you are right I do not need that there. I have removed it